CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.10.6](https://github.com/go-gitea/gitea/releases/tag/v1.10.6) - 2020-03-10
+
+This is a retag version of v1.10.5 and also pin xgo version to go1.13.
+
+WARNNING: 1.10.5 is 100% busted (in the sense that it's actually 1.12).
```

Comment:
```suggestion
WARNING: 1.10.5 is 100% busted (in the sense that it's actually 1.12).
```

Reply:
@guillep2k It seems @jolheiser 's suggestion is better. Just followed his. :)